rule/expr.go

```diff
@@ -515,8 +545,15 @@ func (v *Value) Equal(other *Value) bool {
 	return v.compare(token.EQL, other)
 }
 
-type operander interface {
+// Operander is an interface for managing the operands of an
+// Expr that is an operation.
+type Operander interface {
+	// Operands returns of an operations Operands
```

Comment:
Not sure to understand the comment here.

Reply:
Nor me!  Good catch ;-)